### DISCUSSION TITLE
glob --depth not working? Applying .gitignore rules?

### DISCUSSION THREAD

**Discussion by Schweber** [nushell/nushell#12108] [Q&A] [answered]
I have several `.mp4` on my system but if i run e.g. `glob -DSd 99 *.mp4` from my home directory, i only get the `.mp4` files that are in my cwd but none of the files in subfolders. It seems to me that the `--depth` flag has no effect and no recursion is taking place. I played around with different values, flags, places to search etc. but also a plain `glob --depth 9 .mp4` never showed me results below my cwd.

Am i doing something wrong or is something wrong with the `glob` command? Does `glob` respect .gitignore? If so, i found no flag to turn it off.

**Comment by fdncred** [comment, marked as answer]
It looks to me like it's your glob pattern that is wrong.
![image](https://github.com/nushell/nushell/assets/343840/69fa7390-071c-4081-9a6c-b0ac7c7c022d)

**Reply by Schweber** [reply]
You"re right, thanks!
